packages/ifpack2/src/Ifpack2_Details_MultiVectorLocalGatherScatter.hpp

```diff
@@ -148,7 +195,7 @@ class MultiVectorLocalGatherScatter {
 
   template<typename InView, typename OutView>
   void scatterViewToView(InView& X_in,
```

Comment:
Either pass both Views by value, or pass them both by const reference.  View is a (fancy) pointer, not a container like vector.